app/src/test/java/org/pesmypetcare/mypetcare/controllers/exercise/TestTrGetAllWalks.java

```diff
@@ -0,0 +1,127 @@
+package org.pesmypetcare.mypetcare.controllers.exercise;
+
+import com.google.android.gms.maps.model.LatLng;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.httptools.exceptions.InvalidFormatException;
+import org.pesmypetcare.httptools.utilities.DateTime;
+import org.pesmypetcare.mypetcare.features.pets.Pet;
+import org.pesmypetcare.mypetcare.features.pets.PetRepeatException;
+import org.pesmypetcare.mypetcare.features.pets.events.exercise.walk.Walk;
+import org.pesmypetcare.mypetcare.features.pets.events.exercise.walk.WalkPets;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.usermanager.datacontainers.pet.GenderType;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Daniel Clemente
+ */
+public class TestTrGetAllWalks {
+    private User user;
+    private Walk walk;
+    private Walk walk2;
+    private Pet pet;
+    private Pet pet2;
+    private TrGetAllWalks trGetAllWalks;
+
+    @Before
+    public void setUp() throws PetRepeatException, InvalidFormatException {
+        user = new User("johnDoe", "johndoe@gmail.com", "1234");
+        pet = getDinkyPet("Dinky");
+        pet2 = getDinkyPet("Dinky2");
+        user.addPet(pet);
+        user.addPet(pet2);
+        trGetAllWalks = new TrGetAllWalks();
+
+        List<LatLng> coordinates = Arrays.asList(new LatLng(0.0, 0.0), new LatLng(1.0, 1.0));
+        walk = new Walk("walk", "nice", DateTime.Builder.build(2020, 2, 2, 16, 30, 0),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['2020' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)